config/secrets.yml

```diff
@@ -60,31 +57,16 @@ test:
     google_oauth2:
       enabled: true
 
-integration:
-  <<: *default
-  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
-  sendgrid: <%= !ENV["SENDGRID_USERNAME"].blank? %>
-  smtp_username: <%= ENV["MAILER_SMTP_USER_NAME"] %>
-  smtp_password: <%= ENV["MAILER_SMTP_PASSWORD"] %>
-  smtp_address: <%= ENV["MAILER_SMTP_ADDRESS"] %>
-  smtp_domain: <%= ENV["MAILER_SMTP_DOMAIN"] %>
-  smtp_port: <%= ENV["MAILER_SMTP_PORT"] %>
-  smtp_starttls_auto: false
-  smtp_authentication: "login"
-
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   <<: *default
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   sendgrid: <%= !ENV["SENDGRID_USERNAME"].blank? %>
-  smtp_username: <%= ENV["MAILER_SMTP_USER_NAME"] || ENV["SENDGRID_USERNAME"] %>
-  smtp_password: <%= ENV["MAILER_SMTP_PASSWORD"] || ENV["SENDGRID_PASSWORD"] %>
-  smtp_address: <%= ENV["MAILER_SMTP_ADDRESS"] || "smtp.sendgrid.net" %>
-  smtp_domain: <%= ENV["MAILER_SMTP_DOMAIN"] || "heroku.com" %>
-  smtp_port: <%= ENV["MAILER_SMTP_PORT"] || "587" %>
-  smtp_starttls_auto: false
-  smtp_authentication: "login"
+  smtp_username: <%= ENV["MAILER_SMTP_USER_NAME"] %>
+  smtp_address: <%= ENV["MAILER_SMTP_ADDRESS"] %>
+  smtp_domain: <%= ENV["MAILER_SMTP_DOMAIN"] %>
+  smtp_port: <%= ENV["MAILER_SMTP_PORT"] %>
```

Comment:
this makes this app different from the clean-app without any reason. I wouldn't do this changes